ui/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "BSD",
   "scripts": {
     "ng": "ng",
-    "start": "npm run codegen && ng serve",
```

Comment:
Removing codegen from `start` without removing it from the others -- is this intentional? I'm assuming that `build` will fail from inside the ui container as currently defined (since no swagger file is present).